msb-test.sh

```diff
@@ -8,11 +8,8 @@
 # ./test.sh -d             clean test containers
```

Comment:
not needed? Do msb tests get run while running regular test.sh?

Reply:
MSB tests do run when running test.sh but msb-test.sh is not used anywhere.

Reply:
Removed.